Source/Scene/Cesium3DTileset.js

```diff
@@ -523,6 +524,14 @@ define([
         this.loadSiblings = defaultValue(options.loadSiblings, false);
 
         /**
+         * An array of {@link Plane} used to clip the tileset.
+         *
+         * @type {Plane[]}
+         * @default []
+         */
+        this.clippingPlanes = defaultValue(options.clippingPlanes, []);
```

Comment:
Minor, but here and below we usually would not use `defaultValue` when the second parameter is an allocation.  Instead used `defined` and `?`.

However, couldn't `clippingPlanes` also just be `undefined`?  I'm pretty sure we allow that in other places where `undefined` and empty are treated the same.

If not, please make sure that we throw a `DeveloperError` when it is `undefined`.